src/main/scala/wdlTools/eval/stdlib/BaseStdlib.scala

```diff
@@ -454,7 +519,8 @@ case class StdlibV1(opts: Options, evalCfg: EvalConfig, docSourceUrl: Option[URL
 
   // Array[Array[X]] transpose(Array[Array[X]])
   //
-  private def transpose(args: Vector[V], text: TextSource): V_Array = {
+  // since: draft-2
+  protected def transpose(args: Vector[V], text: TextSource): V_Array = {
```

Comment:
Here and in the polymorphic functions below, note that we can implement them _without_ knowing the types _X_ and _Y. This is a place where type-erasure actually works.